pwndbg/commands/context.py

```diff
@@ -43,6 +43,19 @@ def clear_screen():
                                                   'which context sections are displayed by default (also controls order)')
 
 
+@pwndbg.config.Trigger([config_context_sections])
+def validate_context_sections():
+    valid_values = [context.__name__.replace('context_', '') for context in context_sections.values()]
+    for section in get_context_sections():
+        if section not in valid_values:
+            config_context_sections.value = config_context_sections.default
+            return
```

Comment:
Can we print some hint/warn/error about `valid_values` and that we are switching back to default?

Reply:
it existed once... i swear... weird well the hell it vanished... :open_mouth: